src/components/TeamCard.astro

```diff
@@ -0,0 +1,61 @@
+---
+import Icon from "./Icon.astro";
+export interface Props {
+  name: string;
+  linkedin_url: string;
+  github_username: string;
+}
+
+const { name, linkedin_url, github_username } = Astro.props as Props;
+---
+
+<li class="link-card">
+  <div>
+    <img src={`https://avatars.githubusercontent.com/${github_username}`} alt="PFP" class="pfp" />
```

Comment:
If the `alt` property is set, let's set it to something useful like `Profile Picture` or `${name}'s Profile Picture`